test/ts-app.js

```diff
@@ -12,5 +12,17 @@ t.test('TypeScript app', async t => {
     (await ua.getOk('/hello')).statusIs(200).bodyLike(/Hello TypeScript controller!/);
   });
 
+  await t.test('Query Parameter Parse', async () => {
+    (await ua.getOk('/hello-name', {query: {name: 'User'}})).statusIs(200).bodyLike(/Hello User!/);
+  });
+
+  await t.test('Post JSON Body Parse', async () => {
+    (await ua.postOk('/data', {json: {name: 'User'}})).statusIs(200).jsonIs({greeting: 'Hello User'});
+  });
+
+  await t.test('Get Path Param Parse', async () => {
```

Comment:
Those test descriptions are not proper english.

Reply:
Amended - any preferred format for test descriptions?  I've seen projects where heavy use of "Can..." or "Has a..." or "Throws when...", some describing the operation in detail, and then some just all over the place...

Reply:
No preference, it should just be descriptive enough to be useful.